hcfb-fe/src/app/rescue/rescue.service.ts

```diff
@@ -11,11 +19,21 @@ export class RescueService {
   });
 
   currentRescue = this.rescueSource.asObservable();
-  constructor() { }
-  
+  constructor(private http: HttpClient ) { }
+  options = {headers : {'Content-Type' : 'application/json'}};
+
+  private static _handleError(err: HttpErrorResponse | any) {
+    return throwError(err.message || 'Error: Unable to complete request.');
+ }
+
   changeRescue(rescue: Rescue) {
     this.rescueSource.next(rescue)
+  }
 
+  getRescueLocation(): Observable<any> {
```

Comment:
It'd be nice to be able to have an actual type here 🤔 Ideally, one that's guaranteed to stay up to date with the actual type returned by the backend API.

I think being able to do that well would take a lot of work, though - likely extracting the model to an independent module and porting it to TypeScript.